base/libgit2/blob.jl

```diff
@@ -9,9 +9,12 @@ end
 
 Fetch the *raw* contents of the [`GitBlob`](@ref) `blob`. This is a read-only
 `Array` containing the contents of the blob, which may be binary or may be ASCII
-`String` data.
+`String` data. `rawcontent` will allow the user to load the raw binary data into
+the output `Array` and will not check to ensure it is a valid `String`, so errors
+may occur if the result is passed to functions which expect valid `String` data.
 
-See also [`content`](@ref).
+See also [`content`](@ref), which *will* throw an error the content of the `blob`
+is binary and not valid ASCII.
```

Comment:
Actually I think it shouldn't, the libgit2 docs only refer to "printable characters"